From 6a3d3fa6df953822f8416e3a49a81d28f77b0509 Mon Sep 17 00:00:00 2001
From: qianmenx <qianx.meng@intel.com>
Date: Fri, 17 May 2019 11:31:05 +0800
Subject: [PATCH 1450/1505] media: intel-ipu4: ox03a10: delete unused exposure
 mode

delete unused exposure mode(x3a)

Change-Id: Id49c9255026245d7b6df48d5aeab3c4d177b23eb
Tracked-On: PKT-2588
Tracked-On: #JIIAP-798
Signed-off-by: qianmenx <qianx.meng@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/i2c/crlmodule/crl_ox03a10_common.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/media/i2c/crlmodule/crl_ox03a10_common.h b/drivers/media/i2c/crlmodule/crl_ox03a10_common.h
index f18382582c79..dc7ab02c85bc 100644
--- a/drivers/media/i2c/crlmodule/crl_ox03a10_common.h
+++ b/drivers/media/i2c/crlmodule/crl_ox03a10_common.h
@@ -1016,10 +1016,6 @@ struct crl_ctrl_data_pair ox03a10_ctrl_data_modes[] = {
 	{
 		.ctrl_id = CRL_CID_EXPOSURE_MODE,
 		.data = 1,
-	},
-	{
-		.ctrl_id = CRL_CID_EXPOSURE_MODE,
-		.data = 2,
 	}
 };
 
-- 
2.17.1

